docs/source/operations/projections/ortho.rst

```diff
@@ -37,6 +37,8 @@ around a given latitude and longitude.
           ellipsoid must be forced to a sphere, for example by adding a ``+f=0``
           parameter.
 
+This projection method corresponds to ``ÈPSG:9840``.
```

Comment:
Thanks for adding this! There might be a typo: È instead of E

Reply:
fixed